extensions/security/iprange/src/main/java/org/exist/security/realm/iprange/IPRangeServlet.java

```diff
@@ -24,18 +24,17 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.exist.http.servlets.ExistExtensionServlet;
-import org.exist.security.AbstractRealm;
 import org.exist.security.AuthenticationException;
 import org.exist.security.SecurityManager;
 import org.exist.security.Subject;
 import org.exist.xquery.XQueryContext;
 
-import javax.servlet.ServletConfig;
```

Comment:
from my memories this servlet could and should be removed; the original requester does not use it anymore....